cernopendata/mappings/v7/records/record-v1.0.0.json

```diff
@@ -9,9 +9,61 @@
             "analyzer": "title_analyzer",
             "fields": {
               "exact": {
+                "type": "keyword"
+              }
+            }
+        },
+        "authors": {
+          "properties": {
+            "affiliation": {
+              "type": "text",
+              "fields": {
+                "exact": {
+                  "type": "keyword"
+                }
+              }
+            },
+            "rorid": {
+              "type": "text",
+              "fields": {
+                "exact": {
+                  "type": "keyword"
+                }
+              }
+            },
+            "ccid": {
+              "type": "text",
+              "fields": {
+                "exact": {
                   "type": "keyword"
+                }
+              }
+            },
+            "inspireid": {
+              "type": "text",
+              "fields": {
+                "exact": {
+                  "type": "keyword"
+                }
+              }
+            },
+            "orcid": {
```

Comment:
Works well :+1: However we should look at better tokenizer later, because currently there are too many results found for ORCID searches:

- "0000-0002-2634-4958" ... 1 hit
- 0000-0002-2634-4958 ... 196 hits, where the "proper hit" is on the 5th place only

We should treat "-" in ORCID fields a bit like "/" in title fields... perhaps at the index time.